### PR TITLE
config object with dynamic db name

### PR DESCRIPTION
**Pull request by VladVega**
Here is the postgres code on top of https://github.com/juttle/juttle-sql-adapter-common/pull/46

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,19 +56,18 @@ within Juttle. To do so, use the following format when adding a Postgres configu
 ```json
 {
     "adapters": {
-        "postgres": {
-            "connection": "postgres://postgres@localhost:5432/postgres"
-        }
+        "postgres": [{
```
Comment (davidvgalbraith) [inline README.md:59]: Didn't we say this array would be nested in `instances`?

File: index.js [hunk 2]
```diff
@@ -5,18 +5,50 @@
...
+        var conf = _.clone(singleDBConfig);
+        if (options.db) {
+            conf.db = options.db;
+        }
 
-    if (config.connection) {
-        clientSpecficConfig.knex = require('knex')({
+        var conn = getConnectionProperty(conf);
+
+        return require('knex')({
```
Comment (davidvgalbraith) [inline index.js]: Why have we got the `require` in here instead of at the top?

File: README.md (hunk 1 above)
Comment (VladVega) [inline README.md:59]: Yes but:
1. This `instances` stuff is not implemented yet in the juttle repo. Should I do that? @demmer?
2. I'll change the readme to the correct format when I bump to a juttle version that uses `instances`.

File: index.js (hunk 2 above)
Comment (VladVega) [inline index.js]: moved to the top.

**Comment by davidvgalbraith** [comment]
+1

**Pull request merged**